README.md

```diff
@@ -2,20 +2,22 @@
     <a href="https://www.manim.community/"><img src="https://raw.githubusercontent.com/ManimCommunity/manim/master/logo/cropped.png"></a>
     <br />
     <br />
+    <a href="https://pypi.org/project/manimce/"><img src="https://img.shields.io/pypi/v/manim.svg?style=flat" alt="PyPI Latest Release"></a>
     <a href="http://choosealicense.com/licenses/mit/"><img src="https://img.shields.io/badge/license-MIT-red.svg?style=flat" alt="MIT License"></a>
     <a href="https://www.reddit.com/r/manim/"><img src="https://img.shields.io/reddit/subreddit-subscribers/manim.svg?color=orange&label=reddit" alt="Reddit" href=></a>
+    <a href="https://github.com/psf/black"><img src="https://img.shields.io/badge/code%20style-black-000000.svg" alt="Code style: black">
```

Comment:
Comment, non-blocking: I'm not sure whether we actually need a `black` badge in our readme, but won't block the PR because of it.

Reply:
same idea here, that was inspired by other repos:
https://github.com/pandas-dev/pandas